plugins/dell/fu-plugin-dell.c

```diff
@@ -253,6 +253,11 @@ fu_plugin_dock_node (FuPlugin *plugin, GUsbDevice *device,
 				     component_desc);
 	fu_device_set_id (dev, dock_id);
 	fu_device_set_name (dev, dock_name);
+	if (type == DOCK_TYPE_TB16) {
+		fu_device_set_summary (dev, "A Thunderbolt™ 3 docking station");
+	} else if (type == DOCK_TYPE_WD15) {
+		fu_device_set_summary (dev, "A Display Port docking station");
```

Comment:
Following the style of this PR, this would be better to be "A USB type-C docking station"

Reply:
Agree.